spec/plugin.spec.js

```diff
@@ -286,6 +286,45 @@ describe('CspHtmlWebpackPlugin', () => {
     );
   });
 
+  it('adds meta tag when no template is specified', done => {
+    const webpackConfig = {
+      entry: path.join(__dirname, 'fixtures/index.js'),
+      output: {
+        path: OUTPUT_DIR,
+        filename: 'index.bundle.js'
+      },
+      plugins: [
+        new HtmlWebpackPlugin({
+          filename: path.join(OUTPUT_DIR, 'index.html'),
+          inject: 'body'
```

Comment:
Would you be able to create a new template `with-no-meta-tag.html` and explicitly set it here, using the `template` param? Example above :) 

That way, if other examples change, this one won't start failing

Reply:
Okay I'll add a `with-no-meta-tag.html` test case.

Wouldn't you still want this original test case in addition to what you requested? This test case specifies no template at all, using the automatically generated template from HtmlWebpackPlugin. How would it start failing if other examples changed?

Reply:
No template defaults to an index.html file, so if your changes are able to add the missing meta tag to the specified template here, it should add it to the one with no template too!

I was initially thinking that if HTML Webpack Plugin changed something (like started requiring a template), then this example may start failing